scripts/feedback.js

```diff
@@ -1,10 +1,38 @@
 const fs  = require('fs');
+const github = require('./utils/github');
+const slack = require('./utils/slack');
+// const brain = require('./utils/brain');
 const json = JSON.parse(fs.readFileSync('config.json', 'utf8'));
 
 module.exports = function(robot) {
-    robot.respond(/feedback (.*)/i, function(msg){
-        var today = new Date();
-        feedback = msg.match[1];
+    robot.respond(/ (feedback|f|queja) (.*)/i, msg => {
+        const feedback = msg.match[2];
+        robot.brain.set(msg.message.user.id, feedback);
+        msg.reply(`This is your feedback: ${feedback}, are you ok? y/n`);
+    });
+
+    robot.respond(/(y|Yes|Y|yes)/i, msg => {
```

Comment:
if the regexp is insensitive is not needed then the `yY` & `yes Yes`.

The same for line `25`.